resources/assets/js/app/services/VehicleService.js

```diff
@@ -137,10 +138,35 @@ export const VehicleValidate = (data = {
     return result;
 };
 
+export const VehicleData = {
+    fetchBrand(name) {
+        return simpleRequest.get(`/api/v1/car-brand/`, {
+            params: {
+                search: name,
+                orderBy: 'name'
+            }
+        });
+    },
+
+    fetchModel(name, brandId) {
```

Comment:
this method is quite similar to getModelOptions, keep one of them, you can make 1 method that will provide request, and other method will use response to provide different data preparation